Website/src/rendering/src/assets/css/components/Speakers/FeaturedSpeakers.css

```diff
@@ -1,28 +1,31 @@
-.featured-speakers {
-  .item-grid {
-    .grid-content {
-      @apply grid grid-cols-1;
-      @apply sm:grid-cols-2;
-      @apply lg:grid-cols-3;
+.featured-speakers.item-grid {
+  .grid-content {
+    @apply sm:gap-x-2 sm:gap-y-2;
+    @apply md:gap-x-6 md:gap-y-4;
+    @apply lg:gap-x-8 lg:gap-y-6;
+    @apply xl:gap-x-12 xl:gap-y-8 xl:grid-cols-3;
 
-      .grid-item {
-        @apply sm:mx-2 sm:mb-2;
-        @apply md:mx-6 md:mb-4;
-        @apply lg:mx-8 lg:mb-6;
-        @apply xl:mx-12 xl:mb-8;
+    .grid-item {
+      .item-image {
+        @apply bg-no-repeat bg-orange;
 
-        .item-image {
-          @apply bg-no-repeat bg-orange;
+        background-image: url(../../../img/background-element.svg);
+      }
 
-          background-image: url(../../../img/background-element.svg);
-        }
+      .item-details {
+        @apply text-left p-0;
+      }
+    }
+  }
+}
 
+.section--dark,
+.section--dark-pattern {
```

Comment:
`section--dark-pattern` should not be needed in here because all sections that will define it (only setting the background-image) should also specify `section--dark` (to get the right text colors).

That being said, I noticed that "/sitecore/content/EdgeWebsite/home/speakers/Page Components/Featured Section" is missing the `section--dark` cssClass.

Reply:
why use two classes when we can do with one? also when classNames move to dropdown from a rich text field it will be hard to define multiple classes.

Reply:
`section--dark-pattern` is now only setting the background image to avoid CSS repetition. That is why we also need `section--dark` for the colors when using the pattern.

If this will be a problem, then we could change `sections.css` to set the colors on both --dark, --dark-pattern, --light, and --light-pattern:
```
&--dark,
&--dark-pattern {
  @apply bg-black;
  ...
}

&--light,
&--light-pattern {
  @apply bg-white;
  ...
}
```

If you do so, please review all usage of --dark-pattern and --light-pattern to remove the usage of --dark and --light at the same time.